INSTALL.md

```diff
@@ -106,21 +104,43 @@ NOTES:
 
 3. Configure runtime to use dragonfly:
   https://d7y.io/docs/getting-started/quick-start/kubernetes/
-
-```
-## 2.2 Install Nydus components
 ```
-curl -fsSL -o config-nydus.yaml https://raw.githubusercontent.com/dragonflyoss/Dragonfly2/main/test/testdata/charts/config-nydus.yaml
 
-helm install --wait --timeout 10m --dependency-update --create-namespace --namespace nydus-snapshotter nydus-snapshotter dragonfly/nydus-snapshotter -f config-nydus.yaml
+<!-- markdownlint-restore -->
+
+Check that Dragonfly is deployed successfully:
+
+```shell
+$ kubectl wait po --all -n dragonfly-system --for=condition=ready --timeout=10m
+pod/dragonfly-dfdaemon-gs924 condition met
+pod/dragonfly-manager-5d97fd88fb-txnw9 condition met
+pod/dragonfly-manager-5d97fd88fb-v2nmh condition met
+pod/dragonfly-manager-5d97fd88fb-xg6wr condition met
+pod/dragonfly-mysql-0 condition met
+pod/dragonfly-redis-master-0 condition met
+pod/dragonfly-redis-replicas-0 condition met
+pod/dragonfly-redis-replicas-1 condition met
+pod/dragonfly-redis-replicas-2 condition met
+pod/dragonfly-scheduler-0 condition met
+pod/dragonfly-scheduler-1 condition met
+pod/dragonfly-scheduler-2 condition met
+pod/dragonfly-seed-peer-0 condition met
+pod/dragonfly-seed-peer-1 condition met
+pod/dragonfly-seed-peer-2 condition met
 ```
-In the configuration of `config-nydus.yaml`, there is a `X-Dragonfly-Registry` configuration that needs to be adjusted manually. You need to add the address of the image registry you need to use, such as "https://index.docker.io" corresponding to dockerHub, otherwise P2P acceleration may not take effect.
```

Comment:
It is recommended to keep this part

Reply:
A better way is to link to the document, as the above content is already included in the document. There is no need to redefine it again, and if the document is modified.